### PR TITLE
Update your Description

### PR DESCRIPTION
**Pull request by SauravMiah**
- Please submit all pull requests to branch staging (not master)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # DevSpace
-This is the website used for the DevSpace Technical Conference in Huntsville, Alabama.
+This is the bestwebsite used for the DevSpace Technical Conference in Huntsville, Alabama.
```
Comment (ProfessorTom) [inline README.md:2]: add a space between `best` and `website`, please

**Review by ProfessorTom — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (freestylecoder) [inline README.md:2]: This really isn't the "best" website. Buttering up the reviewer will do you nothing. Please remove.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
 The project is a .Net WebAPI project.
 
 All pull requests must come to the staging branch. This branch automatically deploys to our staging website, which can be found at https://devspaceconf-staging.azurewebsites.net/  I will merge staging into master after everything can get a smoke test.
 
-The staging website uses a backup of the production database. The production database is copied to the staging database at about 3AM every day. If you want to see how that happens, look at the devspace-data project.
+The staging website uses a backup of the production database. The production database is copied to the staging database at about 3am every day. If you want to see how that happens, look at the devspace-data project.
```
Comment (freestylecoder) [inline README.md:8]: AM is correct. However, there should technically be a space between 3 and AM.  So, I'm more than happy to approve your pull request, but it should be "3 AM" and not "3am".

**Review by freestylecoder — CHANGES_REQUESTED** [review]

**Comment by freestylecoder** [comment]
I gave a reasonable amount of time for this to be tweaked. Closed as Hacktoberfest spam.

**Pull request closed without merging**